webapp/lib/controller.dart

```diff
@@ -317,3 +379,9 @@ void saveProjectConfiguration(Map config) {
   model.additionalProjectConfigurationLanguages.removeWhere((l) => languagesAdded.contains(l));
   loadProjectConfigurationView();
 }
+
+void savePackageConfiguration() {}
+
+// Helper Mehods
+
+String get generatePackageId => 'package-${new Uuid().v4().split('-')[0]}';
```

Comment:
Let's make this a normal function rather than a getter. Otherwise when it gets called, it reads really weird since the name is not very getter-like (ie. variable-like)

Reply:
Made into a normal function